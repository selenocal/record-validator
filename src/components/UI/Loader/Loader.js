import React from "react";
import styled from "styled-components";
import Spinner from "../../../assets/images/spinner.gif";

const Loader = () => {
  return (
    <StyledLoader>
      <img src={Spinner} alt="loading" />
    </StyledLoader>
  );
};

const StyledLoader = styled.div`
  position: fixed;
  left: 0px;
  top: 0px;
  width: 100%;
  height: 100%;
  z-index: 9999;
  display: flex;
  justify-content: center;
  align-items: center;
  background: 50% 50% no-repeat rgb(249, 249, 249, 0.5);
`;

export default Loader;
