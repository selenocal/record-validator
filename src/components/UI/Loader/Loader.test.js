import React from "react";
import { shallow } from "enzyme";
import Loader from "./Loader";

describe("<Loader />", () => {
  const wrapper = shallow(<Loader />);

  it("expects render Loader correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
