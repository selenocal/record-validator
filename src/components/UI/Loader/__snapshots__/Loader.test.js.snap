// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> expects render Loader correctly 1`] = `ShallowWrapper {}`;
